ci-cd/Jenkinsfile.groovy

```diff
@@ -15,14 +15,7 @@
  *   along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-def commit = "UNKNOWN"
-def version = "UNKNOWN"
-def dockerRepo = "ghcr.io/overture-stack/maestro"
-pipeline {
-    agent {
-        kubernetes {
-            label 'maestro-executor'
-            yaml """
+String podSpec = '''
```

Comment:
what is podspec?

Reply:
just a variable with the yaml description of the k8s pods specification for the build pods etc.
generally we've had those inline within the jenkins declarative syntax, but I'm just abstracting it to a separate variable, both to isolate syntaxes and for ease of reference when troubleshooting devops concerns.